src/commands/generate/serviceGenerator.ts

```diff
@@ -140,17 +140,22 @@ export function generateService(
 
         endpointDefinition.errors?.forEach(error => {
             // TODO: Get rid of the visitor, because the type (reference) is already known
-            const errorImports: ImportDeclarationStructure[] = IType.visit(
-                {
-                    reference: {
-                        name: `${error.error.name}`,
-                        package: error.error.package,
+            try {
+                const errorImports: ImportDeclarationStructure[] = IType.visit(
+                    {
+                        reference: {
+                            name: error.error.name,
+                            package: error.error.package,
+                        },
+                        type: "reference",
                     },
-                    type: "reference",
-                },
-                importsVisitor,
-            ).map(i => ({ ...i, isTypeOnly: true }));
-            imports.push(...errorImports);
+                    importsVisitor,
+                ).map(i => ({ ...i, isTypeOnly: true }));
+                imports.push(...errorImports);
+            } catch (e) {
```

Comment:
Let's not throw here - we might as well fail loudly if our code is wrong.

Reply:
Done!